content/en/account_management/audit_trail_events.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Audit Trail Events
+kind: documentation
+further_reading:
+- link: "/account_management/audit_trail/"
+  tag: "Documentation"
+  text: "Learn more about Audit Trail"
+---
+
+## Overview
+
+[Datadog Audit Trail][1] records more than 100 types of audit events from across the Datadog platform. These audit events are categorized into different product categories as event names.
+
+#### Platform Events
+- [Access management](#access-management-events)
+- [API request](#api-request-events)
+- [Authentication](#authentication-events)
+- [Dashboard](#dashboard-events)
+- [Integration](#integration-events)
+- [Monitor](#monitor-events)
+- [Notebook](#notebook-events)
+- [OAuth](#oauth-events)
+- [Organization management](#organization-management-events)
+- [Support administration](#support-administration-events)
+
+#### Product-Specific Events
+- [Cloud Security Platform](#cloud-security-platform-events)
+- [Log Management](#log-management-events)
+- [Metrics](#metrics-events)
+- [Real User Monitoring](#real-user-monitoring-events)
+- [Sensitive Data Scanner](#sensitive-data-scanner-events)
+- [Service Level Objectives](#service-level-objectives-slo-events)
+- [Synthetic Monitoring](#synthetics-monitoring-events)
+
+See the [Audit Trail documentation][2] for more information on setting up and configuring Audit Trail.
+
+## Audit Events
+
+### Access management events
+
+| Name        | Description of audit event                                          | Query in audit explorer                           |
+| ----------- | ------------------------------------------------------------------- | --------------------------------------------------|
+| Application key (Service account user) | A user created, modified, or deleted an application key for a service account user. | `@evt.name:"Access Management" @asset.type:application_key` |
+| Authentication methods (Org) | A user modified the allowed authentication methods for an org and what the previous and new values are. | `@evt.name:"Access Management" @asset.type:identity_provider` |
+| Email       | An email is added, disabled, or verified on the Datadog account as a user in the account. | `@evt.name:"Access Management" @asset.type:user` |
+| Role        | A role is modified and what the previous and new permissions are. | `@evt.name:"Access Management" @asset.type:role @action:modified` |
+| Role        | A role is created or deleted in the org. | `@evt.name:"Access Management" @asset.type:role @action:(created OR deleted)` |
```

Comment:
Do these two have the same name? Would it make sense to differentiate them in some way?

Reply:
@urseberry  What about `Role modified` and `Role created or deleted`?

That's what I was thinking originally, but then it didn't seem consistent with the other Access Management events. For example `User's role` isn't `User's role added or deleted`.

Reply:
It looks like the name doesn't actually show up in the app? The names are only used on this docs page? In that case, I think it's all right for them to be somewhat inconsistent. `Role modified` and `Role created or deleted` seem fine to me. Alternately, you can use parentheses, like you did elsewhere. `Role (modified)` and `Role (created or deleted)`.

Reply:
@urseberry Thanks! Yes, the names are only used on the docs page, so that users can easily skim down the table to find what they are looking for. I updated the roles.